Formula/szip.rb

```diff
@@ -1,10 +1,9 @@
 class Szip < Formula
   desc "Implementation of extended-Rice lossless compression algorithm"
   homepage "https://support.hdfgroup.org/HDF5/release/obtain5.html#extlibs"
-  # https://support.hdfgroup.org/ftp/lib-external/szip/2.1.1/src/szip-2.1.1.tar.gz is 403
-  url "ftp://ftp.hdfgroup.org/lib-external/szip/2.1.1/src/szip-2.1.1.tar.gz"
-  mirror "https://dl.bintray.com/homebrew/mirror/szip-2.1.1.tar.gz"
-  sha256 "897dda94e1d4bf88c91adeaad88c07b468b18eaf2d6125c47acac57e540904a9"
+  url "https://support.hdfgroup.org/ftp/lib-external/szip/2.1.1/src/szip-2.1.1.tar.gz"
+  sha256 "21ee958b4f2d4be2c9cabfa5e1a94877043609ce86fde5f286f105f7ff84d412"
+  revision 2
```

Comment:
`1`, not `2`.

Reply:
Thanks — I'll need to put that on a Post-it note! Corrected.

Reply:
I thought we'd made this an audited thing at some point but perhaps I'm misremembering.

Reply:
You're right, audit shows it. It was me, who missed to re-run the audit after this modification.